requirements.txt

```diff
@@ -0,0 +1,4 @@
+alembic==1.5.7
+Django==3.1.7
+psycopg2==2.8.6
+SQLAlchemy==1.3.23
```

Comment:
Please consider updating these to the newest versions before freezing.  In particular, SQLAlchemy would be better at `1.4.0`+ (some new features were added, which are being used)